web_access_rule_buttons/tests/__init__.py

```diff
@@ -0,0 +1,3 @@
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
```

Comment:
```suggestion
```

License is not needed on `__init__.py` files. Moreover it's inconsistent with the module license (AGPL, not LGPL)